internal/osutils/autostart/autostart_linux.go

```diff
@@ -171,16 +102,25 @@ func isEnabled(exec string, opts Options) (bool, error) {
 }
 
 func autostartPath(name string, _ Options) (string, error) {
-	dir, err := prependHomeDir(autostartDir)
+	// do not report ~/.profile as installed, as it would be removed on uninstall.
+	return "", nil
```

Comment:
Just want to make sure we've verified that the `~/.profile` entries for autostart will be removed on uninstall.

Reply:
I didn't check the current calls to this package, but under my last PR the uninstall process called "Disable" as well as removing any entry returned from this method (previously named "InstallPath"). The disable code for the .profile mod was untouched and is here: https://github.com/ActiveState/cli/pull/2456/files#diff-fe1d6fb736a8d64ee905da5161a57eae198ad600fe9e62e48390d0ad205dd88fR68-R81

Reply:
Please rewrite this logic so it's not implicit. Imo any time we're implicitly returning some value that embeds specialized knowledge of consuming code we should revisit what we're doing.

Reply:
We had a discussion which emphasized the need to ensure this method does not have any knowledge about how it will be called. I think I reached that intention by updating the comment. Please let me know if it's sufficient.